packages/components/molecules/f-searchbox/src/services/general.services.js

```diff
@@ -103,9 +103,22 @@ const fullAddressLocalStorageService = {
     }
 };
 
+/**
+ * Extract postcode from address in agreed format
+ * @param {string} address A address string
+ */
+const extractPostcode = address => {
+    try {
+        return address.split(',')[0];
```

Comment:
This seems to pick up the first item in the Array, is this always the address format?

Reply:
The address format can be either of these two:

1. `AR511AR` - If they continue with a suggested postcode, when they don't want to pick their actual address. 
2. `AR511AR, 10 some place, london` - if they pick their address.

In both scenarios they'll end up with just the postcode as it's always a string. :)